### DISCUSSION TITLE
Reverse slider really hard to read when the slider is really short

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#24984] [Ideas] [answered]
I've been enjoying the new argon (pro) skin a lot and I've switched fully to it, there's one thing that bugs me though
On really short "buzz" sliders it's sometimes *really* hard to distinguish them vs a regular note or just a really short slider note
![image](https://github.com/ppy/osu/assets/66184829/fc39392d-2691-4f96-a4c5-a3c3566cb9d7)

On the triangles skin, although not that great either, there's more of an outline so you can notice it easier when reading high ar maps
![image](https://github.com/ppy/osu/assets/66184829/82df9198-83fa-4909-81be-8b1d3085b620)

On the classic skin, the reverse slider indicator shows up on top, making it the easiest to read that there's a short reverse slider
![image](https://github.com/ppy/osu/assets/66184829/3a614419-016e-41ec-8f50-47b4c7ae3eb0)

It would be really nice if there was a slight change to the ends of reverse sliders to make them stand out more, or maybe making the indicator show up on top like on classic or something along those lines

**Comment by Aergwyn** [comment]
And here I thought I'm just bad. I couldn't quite figure out why I'm struggling with those but a look at stable could've helped. Definitely agree that they are a bit tricky if one doesn't know a map by heart.

**Comment by bytenekos** [comment]
Some videos to show this in play
[Beatmap](https://osu.ppy.sh/beatmapsets/1420993#osu/2926947)

Argon

https://github.com/ppy/osu/assets/66184829/7cae9c59-403c-47f3-ba71-8973ac779508




Triangles


https://github.com/ppy/osu/assets/66184829/ac2103d7-93bc-46cc-a29b-b7bdd6843c11


Classic


https://github.com/ppy/osu/assets/66184829/467d39fe-0292-4f41-a1d4-2540a909cd80

**Comment by peppy** [comment]
There's a design to fix this but it's been abandoned for a year after I asked for a fix on it.

Will try again.


https://www.figma.com/file/up6guVobzpc3DZlQxWzVmv/Gameplay-Layout?type=design&node-id=0-1&mode=design&t=y2KC1Vvwsfxyi2Ao-0

![Safari_2023-10-02_at_03 41 41](https://github.com/ppy/osu/assets/191335/ca0b994c-678f-49d2-b7ac-5d94e62cddca)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/24990
